server.go

```diff
@@ -1944,6 +1963,59 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 	return nil
 }
 
+// SetSendCompressor sets a compressor for outbound messages.
+// It must not be called after any event that causes headers to be sent
+// (see _ServerStream_.SetHeader for a complete list). Provided compressor is used when below
+// conditions are met:
+//
+//   - compressor is registered via encoding.RegisterCompressor
+//   - compressor name exists in the client advertised compressor names sent in
+//     grpc-accept-encoding header. Use _ServerStream_.ClientAdvertisedCompressors
```

Comment:
Same here. Sorry for the confusion.  But also this isn't a method on `grpc.ServerStream`, it is a global function in this package.